sshuttle/firewall.py

```diff
@@ -288,3 +313,12 @@ def main(method_name, syslog, ttl):
                 debug1(traceback.format_exc())
             except BaseException:
                 debug2('An error occurred, ignoring it.')
+
+        try:
+            flush_systemd_dns_cache()
+        except BaseException:
+            try:
+                debug1("Error trying to flush systemd dns cache.")
+                debug1(traceback.format_exc())
+            except BaseException:
```

Comment:
Is this really needed? If the debug1 statement fails, I think something is seriously wrong, and we probably shouldn't continue...

Reply:
I found it odd too, but I followed the pattern used in this section of code. There are many similar blocks above it. They should all be fixed if this is unnecessary.